Content.Client/UserInterface/HumanoidProfileEditor.cs

```diff
@@ -43,25 +46,33 @@ public partial class HumanoidProfileEditor : Control
         private readonly OptionButton _preferenceUnavailableButton;
         private readonly List<AntagPreferenceSelector> _antagPreferences;
 
+        private readonly IEntity _previewDummy;
```

Comment:
Should this be deleted when the editor is closed?

Reply:
Also I noticed elsewhere that OnServerDataLoaded is subscribed to but never unsubscribed when it should be as well (I know it wasn't touched as part of the PR but if you get a chance that'd be great).